src/xo-app/settings/remotes/index.js

```diff
@@ -39,6 +42,31 @@ class AbstractRemote extends Component {
     return editRemote(remote, {name})
   }
 
+  _test = () => {
+    const { remote } = this.props
+    testRemote(remote).then(answer => {
+      const title = <span>
+        <Icon icon={answer.success ? 'success' : 'error'} />
+        {' '}
+        {answer.success ? _('remoteTestSuccess') : _('remoteTestFailure')} {remote.name}
```

Comment:
The issue is that words order is language dependent therefore you cannot concatenate a translated string and expect it to work properly.

You can [pass variables](https://github.com/vatesfr/xo-web/blob/next-release/src/xo-app/jobs/scheduling/index.js#L162) to the [translation](https://github.com/vatesfr/xo-web/blob/next-release/src/common/intl/messages.js#L172).